cves/CVE-2017-7234.yml

```diff
@@ -282,7 +281,10 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: The keep it simple principle seems to be the most applicable to this
+    vulnerability because the additional functionality of redirect, in addition 
+    to more complicated sanitization attributed to the inability of the unit
+    tests to catch the vulnerability. 
```

Comment:
That added functionality might have been required at some point.  Perhaps the mistake was insufficient testing?

Reply:
In the release docs, a developer said "The view no longer does any redirects as they don't provide any known, useful functionality". But you're right, they could have been added a long time ago and when it was removed, never went back to remove it.